### PR TITLE
Added Linux Tests to T1069.002, T1087.002, T1136.002

### PR DESCRIPTION
**Pull request by kevinmstapleton**
**Details:**
Added ldapsearch implementation to T1069.002 and T1087.002 so that the tests support Linux as well. Added ldapadd implementation to T1136.002.

**Testing:**
Local testing using powershell, and an atomic test runner.

some example results:

## T1069.002:
****** EXECUTOR RESULTS ******
\# extended LDIF
\#
\# LDAPv3
\# base <CN=Users,DC=-------,DC=----> with scope subtree
\# filter: (objectClass=group)
\# requesting: dn
\#

\# Domain Computers, Users, ---------
dn: CN=Domain Computers,CN=Users,DC=-----,DC=-----

## T1087.002:
****** EXECUTOR RESULTS ******
\# extended LDIF
\#
\# LDAPv3
\# base <CN=Users,DC=*********,DC=*******> with scope subtree
\# filter: (objectclass=*)
\# requesting: dn
\#

\# Users, ***********
dn: CN=Users,DC=*****,DC=******

\# AccountNum76, Users, *********
dn: CN=AccountNum76,CN=Users,DC=*****,DC=****

## T1136.002 --index 3
*In our testing, we got a server response of refusing to add admin user, which was expected. With the server being configured to allow an admin user to be added, this test should add an admin user and then remove it*

## T1136.002 --index 4
****** EXECUTOR RESULTS ******
ldapadd -h **(host)** -p 389 -x -D ********** -w **(PASSWORD)**-f tempadmin.ldif
adding new entry "cn=Atomic User, cn=Users,dc=*****,dc=****"

**Associated Issues:**
No linux tests present in these tests.

### CODE REVIEW

File: atomics/T1059.002/T1059.002.md [hunk 1]
```diff
@@ -1,46 +0,0 @@
-# T1059.002 - Command and Scripting Interpreter: AppleScript
-## [Description from ATT&CK](https://attack.mitre.org/techniques/T1059/002)
-<blockquote>Adversaries may abuse AppleScript for execution. AppleScript is a macOS scripting language designed to control applications and parts of the OS via inter-application messages called AppleEvents.(Citation: Apple AppleScript) These AppleEvent messages can be sent independently or easily scripted with AppleScript. These events can locate open windows, send keystrokes, and interact with almost any open application locally or remotely.
```
Comment (cyberbuff) [inline atomics/T1059.002/T1059.002.md:3]: Thanks for your first contribution. Welcome to the project 😄 Can you undelete this file ?

**Review by cyberbuff — CHANGES_REQUESTED** [review]

File: atomics/T1059.002/T1059.002.md (hunk 1 above)
Comment (kevinmstapleton) [reply atomics/T1059.002/T1059.002.md]: My apologies, my antivirus really doesn't like that file (there seems to be some bash scripts inside of the markdown). Should be good to go! Also, thanks for the warm welcome, glad to be contributing!

Comment (cyberbuff) [reply atomics/T1059.002/T1059.002.md]: Haha no worries. Thanks for the quick fix.

File: atomics/T1136.002/T1136.002.yaml [hunk 2]
```diff
@@ -84,3 +84,99 @@ atomic_tests:
       cmd /c "net user #{username} /del >nul 2>&1"
     name: powershell
     elevation_required: false  # Requires a user to be a Domain Admin!
+- name:
```
Comment (cyberbuff) [inline atomics/T1136.002/T1136.002.yaml:87]: Can you add a name for the atomic here ?

**Review by cyberbuff — CHANGES_REQUESTED** [review]

File: atomics/T1069.002/T1069.002.yaml [hunk 3]
```diff
@@ -242,4 +242,43 @@ atomic_tests:
...
+  - description: |
+      Packages sssd-ad sssd-tools realmd adcli installed and realm available, ldapsearch
+    prereq_command: |
+      which ldapsearch
+    get_prereq_command: |
+      echo missing ldapsearch command; exit 1
+  executor:
+    elevation_required: false
+    command: |
+      ldapsearch -h #{domain_controller} -p 389 -x -D #{user} -w #{password} -b "CN=Users,DC=#{domain},DC=#{top_level_domain}" "(objectClass=group)" -s sub -a always -z 1000 dn
```
Comment (cyberbuff) [inline atomics/T1069.002/T1069.002.yaml:283]: Can you mention which linux distribution which you are using ? I tried with centos and ubuntu. I am getting an error 
Unknown option for -h and -p as well

Comment (kevinmstapleton) [reply atomics/T1069.002/T1069.002.yaml:283]: Ah, looks like -h and -p are deprecated for -H instead. I'll do a quick rewrite.

I used Ubuntu, would that be placed in the description?

Comment (cyberbuff) [reply atomics/T1069.002/T1069.002.yaml:283]: No worries. I just wanted to confirm so that I can run atomics on it. Thanks again.

Comment (kevinmstapleton) [reply atomics/T1069.002/T1069.002.yaml]: Hey, just made that change from -h -p to -H. Sorry it took a while, got caught up in other issues. Mind taking a look?

Thanks

**Comment by cyberbuff** [comment]
Thanks for your valuable contribution. Keep it up

**Pull request merged**